src/Security/test/DataProtection.Redis.Test/RedisDataProtectionBuilderExtensionsTest.cs

```diff
@@ -44,8 +44,13 @@ public async Task Stores_session_state_in_Redis()
             };
         });
 
+        builder.Services.AddSession(options =>
+        {
+            options.Cookie.HttpOnly = true;
+            options.Cookie.IsEssential = true;
```

Comment:
These turned out to be essential and not excessive?

Reply:
It's recommended practice. `HttpOnly` means that JavaScript can't read it, `IsEssential` makes it work with GDPR.